docs/concepts/Dependency-Resolution.md

```diff
@@ -17,7 +17,7 @@ When multiple packages have the same dependency, then the same package ID can ap
 
 ## Dependency resolution with PackageReference
 
-When installing packages into projects using the PackageReference format, NuGet adds references to a flat package graph in the appropriate file and resolves conflicts ahead of time. This process is referred to as *transitive restore*. Reinstalling or restoring packages is then a process of downloading the packages listed in the graph, resulting in faster and more predictable builds. You can also take advantage of wildcard (floating) versions, such as 2.8.\*, avoiding expensive and error prone calls to `nuget update` on the client machines and build servers.
+When installing packages into projects using the PackageReference format, NuGet adds references to a flat package graph in the appropriate file and resolves conflicts ahead of time. This process is referred to as *transitive restore*. Reinstalling or restoring packages is then a process of downloading the packages listed in the graph, resulting in faster and more predictable builds. You can also take advantage of floating versions, such as 2.8.\*,  to avoid modifying the project to use the latest version of a package.
```

Comment:
Should we formalize the noun "Floating Versions", or bold/italic/etc to emphasize it?

Reply:
in my opinion, no.